JASP-Desktop/widgets/listmodellabelvalueterms.cpp

```diff
@@ -0,0 +1,148 @@
+//
+// Copyright (C) 2013-2020 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+#include "listmodellabelvalueterms.h"
+#include "log.h"
+
+ListModelLabelValueTerms::ListModelLabelValueTerms(QMLListView* listView)
+	: ListModelTermsAvailable(listView)
+{
+	readModelProperty(listView);
+}
+
+QVariant ListModelLabelValueTerms::data(const QModelIndex &index, int role) const
+{
+	int row = index.row();
```

Comment:
You should probably check for `< 0` in case bogus indices are sent. Right now this would lead to an error